[Android] Fatal Exception: java.lang.IllegalArgumentException

I am using `@react-native-community/blur": "^4.2.0"`

Getting below crash issue in Firebase Crashlytics.

![image](https://user-images.githubusercontent.com/47143115/186593506-70aec7ef-c61a-4f23-8aaa-b78b6b86474c.png)

```
Fatal Exception: java.lang.IllegalArgumentException: Software rendering doesn't support drawRenderNode
       at android.graphics.Canvas.drawRenderNode(Canvas.java:2317)
       at eightbitlab.com.blurview.RenderEffectBlur.render(RenderEffectBlur.java:2)
       at eightbitlab.com.blurview.PreDrawBlurController.draw(PreDrawBlurController.java:56)
       at eightbitlab.com.blurview.BlurView.draw(BlurView.java:2)
       at android.view.View.draw(View.java:23770)
       at android.view.ViewGroup.drawChild(ViewGroup.java:5355)
       at com.facebook.react.views.view.ReactViewGroup.drawChild(ReactViewGroup.java:23)
       at android.view.ViewGroup.dispatchDraw(ViewGroup.java:5112)
       at com.facebook.react.views.view.ReactViewGroup.dispatchDraw(ReactViewGroup.java:3)
```

> Task :react-native-community_blur:compileDebugJavaWithJavac FAILED
/Users/bayram/Desktop/Sorsware/omuzreact/node_modules/@react-native-community/blur/android/src/main/java/com/reactnativecommunity/blurview/BlurViewManagerImpl.java:35: error: package WindowManager does not exist
        WindowManager.LayoutParams.FLAG_HARDWARE_ACCELERATED,
                     ^
/Users/bayram/Desktop/Sorsware/omuzreact/node_modules/@react-native-community/blur/android/src/main/java/com/reactnativecommunity/blurview/BlurViewManagerImpl.java:36: error: package WindowManager does not exist
        WindowManager.LayoutParams.FLAG_HARDWARE_ACCELERATED
                     ^
/Users/bayram/Desktop/Sorsware/omuzreact/node_modules/@react-native-community/blur/android/src/main/java/com/reactnativecommunity/blurview/BlurViewManagerImpl.java:34: error: cannot find symbol
      getWindow().setFlags(
      ^
  symbol:   method getWindow()
  location: class BlurViewManagerImpl
Note: /Users/bayram/Desktop/Sorsware/omuzreact/node_modules/@react-native-community/blur/android/src/main/java/com/reactnativecommunity/blurview/BlurViewManagerImpl.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
3 errors

FAILURE: Build completed with 2 failures.

1: Task failed with an exception.

could not build the debug app 

```

> Task :react-native-community_blur:compileDebugJavaWithJavac FAILED
/Users/bayram/Desktop/Sorsware/omuzreact/node_modules/@react-native-community/blur/android/src/main/java/com/reactnativecommunity/blurview/BlurViewManagerImpl.java:35: error: package WindowManager does not exist
        WindowManager.LayoutParams.FLAG_HARDWARE_ACCELERATED,
                     ^
/Users/bayram/Desktop/Sorsware/omuzreact/node_modules/@react-native-community/blur/android/src/main/java/com/reactnativecommunity/blurview/BlurViewManagerImpl.java:36: error: package WindowManager does not exist
        WindowManager.LayoutParams.FLAG_HARDWARE_ACCELERATED
                     ^
/Users/bayram/Desktop/Sorsware/omuzreact/node_modules/@react-native-community/blur/android/src/main/java/com/reactnativecommunity/blurview/BlurViewManagerImpl.java:34: error: cannot find symbol
      getWindow().setFlags(
      ^
  symbol:   method getWindow()
  location: class BlurViewManagerImpl
Note: /Users/bayram/Desktop/Sorsware/omuzreact/node_modules/@react-native-community/blur/android/src/main/java/com/reactnativecommunity/blurview/BlurViewManagerImpl.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
3 errors

```

--- a/android/src/main/java/com/reactnativecommunity/blurview/BlurViewManagerImpl.java
+++ b/android/src/main/java/com/reactnativecommunity/blurview/BlurViewManagerImpl.java
@@ -29,7 +29,7 @@ class BlurViewManagerImpl {
       .getDecorView();
     ViewGroup rootView = decorView.findViewById(android.R.id.content);
     Drawable windowBackground = decorView.getBackground();
-    if (Build.VERSION.SDK_INT >= 31) {
+    if (Build.VERSION.SDK_INT >= 31 && blurView.isHardwareAccelerated()) {
       blurView
         .setupWith(rootView, new RenderEffectBlur())
         .setFrameClearDrawable(windowBackground)

